package-overrides/github/CodeSeven/toastr@2.1.1.json

```diff
@@ -0,0 +1,22 @@
+{
+  "main": "toastr",
+  "registry": "github",
+  "format": "global",
+  "dependencies": {
+    "jquery": "github:components/jquery",
+    "css": "github:systemjs/plugin-css"
+  },
+  "files": [
+    "build/toastr.min.css",
+    "toastr"
+  ],
```

Comment:
Perhaps make this `["toastr.js", "build"]`? Also `toastr` without an extension in theory shouldn't work here.